### ISSUE TITLE
Convert from dev branch to master for release

### ISSUE THREAD

**Issue by kslong** [sirocco-rt/sirocco#621]
The main branch should be master instead of dev in the release so that we are following the normal convention.

Note that we need a philosophy going forward.  One is that we just update master on releases.  This would work well if we were a large software team.  The other possibility is that we use the master branch as the main branch (like we do dev now), and use tags or something to describe releases.  I think I favor this, as we are a small group and are unlikely to be very good about releases.

(We could also use a better understood plan regarding versions.)

**Comment by jhmatthews** [comment]
Default branch is changed now - @Higginbottom just be aware of this since you weren't in the telecon. 

Strategy is to push to to dev and merge to master when we reach checkpoints/releases.

We need to decide on a pre-frame commit checkpoint to merge into master for the moment - I had a quick look but couldn't be sure when would be best. It would also be good to associate a tag with this commit (e.g. v84e). Once this is merged we can close this issue.

**Comment by jhmatthews** [comment]
This commit seems reasonable: https://github.com/agnwinds/python/commit/598949b1efaa25b927a04c62256cad3329410d3d

It's 84e but with a few bug fixes from Knox, but before Nick's charge exchange stuff and the majority of frame changes. I'm going to tag that version as 84e, and merge into master and close this issue.

**Comment by jhmatthews** [comment]
Closed by #733
